projects/demo/src/modules/components/editor-new/examples/2/smiles-tool/smiles-tool.component.ts

```diff
@@ -33,7 +33,8 @@ export class ExampleSmilesToolComponent {
             .getOriginTiptapEditor()
             .chain()
             .focus()
-            .insertContent(`${smile} `)
+            .insertContent(`<p data-type="emoji">${smile}</p>`)
```

Comment:
may be without extra extension and change only this line to:
```ts
// comment why we do in the such way
.insertContent(`<p style="display: inline">${smile}</p>`)
```
?

Reply:
I was trying 
but in the end it was inserted into DOM only `<p>${smile}</p>`